docs/ldml/tr35-general.md

```diff
@@ -2687,14 +2687,14 @@ Feature that classifies nouns in classes. This is grammatical gender, which may
 
 Feature that encodes the syntactic (and sometimes semantic) relationship of a noun with the other constituents of the sentence. (adapted from [linguistics-ontology.org/gold/2010/CaseProperty](http://linguistics-ontology.org/gold/2010/CaseProperty))
 
-##### Example
+###### Example
```

Comment:
Are these meant to be captions? How about the ones in 15.1?

Reply:
I made it a caption on the code block.